apps/vanilla/src/helpers/guards.ts

```diff
@@ -0,0 +1,36 @@
+import { Status } from '@js-camp/core/models/anime';
+import { HttpErrorDto } from '@js-camp/core/dtos/httpError.dto';
+
+import { SortField, SortOrdering } from '../types/paginationSettings';
+
+/**
+ * Check if the error is of type HttpErrorDto.
+ * @param error Some error.
+ */
+export function isHttpErrorDto(error: unknown): error is HttpErrorDto {
+  return (error as HttpErrorDto).detail !== undefined;
+}
+
+/**
+ * Checks if a value is of a enum Status.
+ * @param value Some string.
+ */
+export function isStatus(value: string): value is Status {
+  return Object.values(Status).includes(value as Status);
+}
```

Comment:
Please, be consistent with your docs. Check it everywhere.
```suggestion
/**
 * Checks if the error is of type HttpErrorDto.
 * @param error Some error.
 */
export function isHttpErrorDto(error: unknown): error is HttpErrorDto {
  return (error as HttpErrorDto).detail !== undefined;
}

/**
 * Checks if a value is of a enum Status.
 * @param value Some string.
 */
export function isStatus(value: string): value is Status {
  return Object.values(Status).includes(value as Status);
}
```